RedHatInsights/insights-results-aggregator
# [CCXDEV-3976] add haberdasher configuration

# Description

this PR enables the insights-result-aggregator and the insights-result-db-writer to run with haberdasher logging

## Type of change


- New feature (non-breaking change which adds functionality)

## Testing steps
build the docker image and than run it, verify that your first log line is like so:
`2021/04/13 13:54:07 Initializing haberdasher.`

and no subsequent errors are shown
